CODE_OF_CONDUCT.md

```diff
@@ -1,10 +1,83 @@
-# BIDS Code of Conduct
 
-## Our Pledge
+# Brain Imaging Data Structure Community Code of Conduct
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+## Table of contents:
 
-## Our Standards
+### [The BIDS Code of Conduct](#coc)
+[Our Pledge](#our-pledge)
+
+[Our Standards](#our-standards)
+
+[Our Responsibilities](#our-responsibilities)
+
+[Scope](#scope)
+
+### [Code of Conduct Enforcement](#enforcement)
+
+[Community Member Procedure For Reporting Code of Conduct Incidents](#community-member-procedure-for-reporting-code-of-conduct-incidents)
+
+[General BIDS reporting procedure](#general-bids-reporting-procedure)
+
+[Report data](#report-data)
+
+[Confidentiality](#confidentiality)
+
+[Report Handling Procedure](#report-handling-procedure)
+
+[Following up with reporters](#following-up-with-reporters)
+
+[Conflict of Interest](#conflict-of-interest)
+
+[Code of Conduct Enforcement Procedure](#code-of-conduct-enforcement-procedure)
+
+[Summary of processes](#summary-of-processes)
+
+[Acknowledge the report](#acknowledge-the-report)
+
+[Evaluating a report](#evaluating-a-report)
+
+[Jurisdiction](#jurisdiction)
+
+[Impact](#impact)
+
+[Risk](#risk)
+
+[Propose a behavioral modification plan](#propose-a-behavioral-modification-plan)
+
+[Propose consequences](#propose-consequences)
+
+[Work group vote](#work-group-vote)
+
+[Administrators/moderators approval](#administrators/moderators-approval)
+
+[Follow up with the reported person](#follow-up-with-the-reported-person)
+
+[Decide further responses](#decide-further-responses)
+
+[Follow up with the reporter](#follow-up-with-the-reporter)
+
+[Documentation and Privacy policies](#documentation-and-privacy-policies)
+
+[Committee online discussion](#committee-online-discussion)
+
+[Shared documentation](#shared-documentation)
+
+[Status Spreadsheet](#status-spreadsheet)
+
+[Report Documentation](#report-documentation)
+
+[Ongoing Documentation of Event and Community Bans](#ongoing-documentation-of-event-and-community-bans)
+
+[Privacy Concerns](#privacy-concerns)
+
+[Reference](#reference)
+
+## Code of Conduct {#coc}
```

Comment:
Style point: These `{#anchor}`s are being shown but not linked. This does not seem to be a part of GitHub-flavored markdown.

There is an outline available on all Markdown documents by clicking the button in the top right of the page:

![image](https://github.com/user-attachments/assets/8e003f4a-5691-4ecd-bb37-fd84463aefc1)